### DISCUSSION TITLE
ssl.https redirect + disable certificate verification support

### DISCUSSION THREAD

**Discussion by dsnr-coder** [Kong/kong#9201] [Help] [answered]
Hi,

On hitting the below api in the browser, post couple of redirects, I am getting response as 200. 
Using the below piece of lua code made the same https api call, getting response as timeout...
Went through couple of articles but didn't find any solution that can help for my use case...

Could you please help me, any help would be greatly appreciated...

local path="https://localhost:8443/adminAPI

 local res, code, response_headers, status = https.request
  {
    url = path,
    method = "GET",
    sink = ltn12.sink.table(response_body)
  }

**Comment by Tieske** [comment, marked as answer]
that code is from `luasocket`. Do not use that. Use the `lua-resty-http` library instead. It is shipped with Kong already.

**Reply by dsnr-coder** [reply]
Thanks a lot for your help @Tieske it worked
